src/foundation/Icon/Svg/Twitter.svg

```diff
@@ -0,0 +1,3 @@
+<svg viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Looks like there are changes in this PR which are not related to the topic

Reply:
I needed to rebase and had conflicts. Guess those commits are added to the history then.

Reply:
The commits might be still there, but the changes against master here should disappear again.

Reply:
Just noticed I made a bit of a mess, fixing it right now, sorry 😬

Reply:
No problem. Now it looks good.